development/common/config/nginx.conf.template

```diff
@@ -61,6 +61,12 @@ http {
     location = /api/v1/alerts {
       proxy_pass      http://${ALERT_MANAGER_HOST}$request_uri;
     }
+    location = /api/v1/grafana/config {
+      proxy_pass      http://${ALERT_MANAGER_HOST}$request_uri;
+    }
+    location = /api/v1/grafana/state {
+      proxy_pass      http://${ALERT_MANAGER_HOST}$request_uri;
+    }
```

Comment:
this is the local development nginx config. Will it make sense to expose these endpoints on the mimir helm chart's nginx as well? Will it make sense for on-premise Grafana installations to use these endpoints? Or maybe it's too early to tell?

Reply:
I'd say it's definitely too early to tell right now. The endpoints don't do anything but accept the configuration, and there's a lot of work to do before it can be useful. I'd like to keep this as obfuscated as possible until we have tested the final state + migration. 

I have added a task in https://github.com/grafana/alerting-squad/issues/566 to update the helm chart eventually.